fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/ConstantObjectValue.java

```diff
@@ -83,6 +84,12 @@ public Set<CorrelationIdentifier> getCorrelatedToWithoutChildren() {
         return Set.of();
```

Comment:
Can you please make this a `ImmutableSet.of()`. You didn't change it here, I just stumbled over it.

Reply:
ok, I will do as part of solving #2483